specification/cognitiveservices/data-plane/Language/preview/2021-07-15-preview/questionanswering-authoring.json

```diff
@@ -861,9 +861,82 @@
           }
         }
       }
+    },
+    "/query-knowledgebases/projects/{projectName}/feedback": {
+      "post": {
+        "summary": "Update Active Learning feedback.",
+        "operationId": "QuestionAnsweringProjects_ActiveLearningFeedback",
```

Comment:
The token after the underscore should match the action intended, which here is a POST to create new feedback. These become the method names in the clients, so something like `AddFeedback` might make more sense here.